lib/fleiss/backend/active_record/concern.rb

```diff
@@ -20,7 +20,7 @@ module ClassMethods
           def wrap_perform(&block)
             connection_pool.with_connection(&block)
           rescue ::ActiveRecord::StatementInvalid
-            ::ActiveRecord::Base.clear_active_connections!
+            ::ActiveRecord::Base.clear_all_connections!
```

Comment:
This is the right thing to do (clear_ALL instead of clear_ACTIVE).

Bad news is that it's suppressed in `Fleiss::Worker.perform` - error is rescued (just logged) in `&block` that's passed here, just noticed.

I'll address this separately.